test/unit/config.js

```diff
@@ -0,0 +1,3 @@
+const host = Ti.Platform.osname === 'android' ? '10.0.2.2' : 'localhost';
+const port = '3210';
+module.exports.url = `http://${host}:${port}`;
```

Comment:
This file is not used anywhere so it can be removed.

Reply:
All suggested changes are done and PR updated for same.